test/views/EditSetViewSpec.scala

```diff
@@ -49,12 +53,17 @@ class EditSetViewSpec extends BaseSpec {
         editSet.entries.size,
         1
       )
+
+      when(mockGovukTable.apply(Table(anyList))).thenReturn(getEditSetTable())
+      when(mockGovukPagination.apply(Pagination())).thenReturn(any)
       val editSetHtml: Html = editSetInstance(user, title, heading, editSetPage, EditSetEntryRowOrder.defaultOrder)(
         Helpers.stubMessages(),
         CSRFTokenHelper.addCSRFToken(FakeRequest())
       )
 
-      val document = asDocument(editSetHtml)
+      contentType(editSetHtml) mustBe "text/html"
+      println(editSetHtml)
+      val document = asDocument(Helpers.contentAsString(editSetHtml))
       document must haveTitle(title)
```

Comment:
We are asserting some of the aspects of the generated view.

But not nearly to the extent we are in the respective integration test (`EditSetISpec`).

Should we be?

If not, what subset should we be asserting?

Reply:
The task in this PR is to replace the existing integration tests with unit tests. To add additional scenarios from `EditSetISpec` like ordering would mean testing the controller, which has already been done in the `EditSetControllerSpec`

Reply:
I hadn't meant that we are now missing scenarios - I understand that the current task is only to switch to a unit test - but rather possible assertions of the view's contents.

If you compare the assertions we do in `EditSetISpec`, such as [here](https://github.com/nationalarchives/ctd-omega-editorial-frontend/blob/c57298a4d0990e52c5b3daef0ef774f1353d9d6e/it/EditSetISpec.scala#L16-L16), you can see that there are fewer here.

But I'm not sure if this is outside of scope for this ticket.

@rwalpole  can decide that.

Reply:
Actually, now that I see that @rwalpole has approved this PR, he must be OK with it.

So resolving.